framework/src/problems/FEProblemBase.C

```diff
@@ -177,7 +177,7 @@ FEProblemBase::validParams()
   /// 2. _blocks.size() > 0 and no coordinate system was specified, then the whole domain will be XYZ.
   /// 3. _blocks.size() > 0 and one coordinate system was specified, then the whole domain will be that system.
   params.addDeprecatedParam<std::vector<SubdomainName>>(
-      "block", "Block IDs for the coordinate systems", "Please use 'Mesh/block' instead");
+      "block", "Block IDs for the coordinate systems", "Please use 'Mesh/coord_block' instead");
```

Comment:
Are GlobalParams still applied to deprecated params? If so then this PR will not directly fix my issues, but once the deprecated parameter is removed it should